tele_bot/main.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+"""
+Bot to give access to vpn configuration
+
+Ships in docker container
+"""
+
+import logging
+import os
+import userdata
+import wgconf
+
+from telegram import ForceReply, Update, Bot
+from telegram.ext import Application, CommandHandler, ContextTypes, MessageHandler, filters
+from telegram.constants import ParseMode
+
+
+# Enable logging
+logging.basicConfig(
+    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s", level=logging.INFO
+)
+logger = logging.getLogger(__name__)
+
+
+async def send_confirmation_request(bot: Bot, username: str, user_id: int) -> None:
+    admin_ids = userdata.get_admin_ids()
+    for admin_id in admin_ids:
+        await bot.send_message(
+                admin_id, rf"@{username} requests config\. `/accept {user_id}`",
+                parse_mode=ParseMode.MARKDOWN_V2)
+
+
+# Start command - register user and send confirmation request to admins
+async def start(update: Update, context: ContextTypes.DEFAULT_TYPE) -> None:
+    """Send a message when the command /start is issued."""
+    user = update.effective_user
+    data_path, is_admin = userdata.get_user_data(user.id)
+    text = "User already registred, waiting approval from admins"
+    if data_path is not None:
+        text = f"User already exists, type /get_config to get configuration"
+    if data_path is None and is_admin is None:
```

Comment:
`is_admin` - why _is_ if it can be  `None`, I think it is admin id or tocken, change to admin if it is

Reply:
I think it would be a temporary solution and in future we should make a user's class and operate with it. It would have appropriate method like `exists` or `is_admin`. For now i think checking if `is_admin is None` is acceptable.